### DISCUSSION TITLE
Is it easy to port Objeck to new platforms?

### DISCUSSION THREAD

**Discussion by ghost** [objeck/objeck-lang#339] [General]
For example, FreeBSD or Haiku OS. I'm just curious.

**Comment by objeck** [comment]
Porting is possible, as the original code was written for 32-bit Intel OS X targets. 

I suspect it is not too hard to part Objeck with other x64 and ARM64 targets. The 32-bit Intel (IA-32) and ARM (ARMv7) JIT compilers were removed however I suspect they are still functional. The biggest challenge is testing and maintaining multiple targets.
